lib/foreman_openscap/helper.rb

```diff
@@ -10,8 +10,24 @@
 
 module ForemanOpenscap::Helper
   def self.get_asset(cname, policy_id)
-    asset = Host.find_by_name!(cname).get_asset
+    asset = find_host_by_name_or_uuid(cname).get_asset
     asset.policy_ids += [policy_id]
     asset
   end
+
+  private
+
+  def self.find_host_by_name_or_uuid(cname)
+    if defined?(Katello::System)
```

Comment:
As all of this project... :)
Some were created in https://github.com/shlomizadok/foreman_openscap/tree/host_form_scap_policy_proxy/test, but this branch is still work in progess